src/main/java/cs361/battleships/models/Submarine.java

```diff
@@ -7,10 +7,10 @@
 
 public class Submarine extends Ship {
 
-	public Submarine() {
+	public Submarine(String kind) {
```

Comment:
This is probably a good place to restore the proper name and begin storing the submerged option as it's own variable so it's easier to access by other functions